app/controller/sdl/Abstract/Controller.js

```diff
@@ -1549,5 +1549,20 @@ SDL.SDLController = Em.Object.extend(
       }
       return title;
     },
+
+    imageModeChanged: function() { 
+      if (SDL.SDLController.model) {
+        SDL.SDLController.model.setMode(SDL.SDLModel.data.imageMode);
+        length=SDL.OptionsView.commands.items.length;
```

Comment:
@YarikMamykin 
```suggestion
        const length=SDL.OptionsView.commands.items.length;
```

Reply:
@AKalinich-Luxoft changed in https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/800bde30ba931dbfec1d5e9fc1939c1fe7836909